DirectProgramming/DPC++FPGA/Tutorials/Features/private_copies/src/private_copies.cpp

```diff
@@ -0,0 +1,211 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <math.h>
+
+#include <CL/sycl.hpp>
+#include <CL/sycl/INTEL/fpga_extensions.hpp>
+#include <array>
+#include <iomanip>
+#include <iostream>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;

```

Comment:
Please remove this `using namespace sycl;` and use explicit `sycl::` 
We are trying to make this change for all new samples